docs/google/specification.md

```diff
@@ -113,8 +108,7 @@ engine = cg.Engine(project_id='your_project_id',
                    proto_version=cirq_google.ProtoVersion.V2)
 
 # Replace the processor id to get the device with that id.
-device = engine.get_processor('processor_id').get_device(
-  gate_sets=[cg.gate_sets.SQRT_ISWAP_GATESET])
+device = engine.get_processor('processor_id').get_device()
 
 q0, q1 = cirq.LineQubit.range(2)
 
```

Comment:
I don't think we allow device in Circuit anymore.  So this should use device to validate circuit (sorry know this wasn't part of your change).

Reply:
Fixed, thank you for noticing that!